Application/EdFi.Ods.Api/Container/Modules/ContextStorageModule.cs

```diff
@@ -15,14 +15,14 @@ public class ContextStorageModule : Module
         protected override void Load(ContainerBuilder builder)
         {
             // Primary context storage for ASP.NET web applications
-            builder.RegisterType<HttpContextStorage>().As<IContextStorage>().AsSelf();
+            //builder.RegisterType<HttpContextStorage>().As<IContextStorage>().AsSelf();
 
             // Secondary context storage for background tasks running in ASP.NET web applications
             // Allows selected context to flow to worker Tasks (see IHttpContextStorageTransferKeys and IHttpContextStorageTransfer)
             builder.RegisterType<CallContextStorage>().As<IContextStorage>().AsSelf();
 
             // Features to transfer context from HttpContext to the secondary storage in ASP.NET applications
-            builder.RegisterType<HttpContextStorageTransfer>().As<IHttpContextStorageTransfer>();
+            builder.RegisterType<IHttpContextStorageTransfer>();
```

Comment:
This look wrong, should it not be builder.RegisterType<HttpContextStorageTransfer>().As<IHttpContextStorageTransfer>();

Reply:
Fixed